site/committers.html

```diff
@@ -613,9 +629,9 @@ <h4>Merge Script</h4>
 
 <p>If you want to amend a commit before merging – which should be used for trivial touch-ups – 
 then simply let the script wait at the point where it asks you if you want to push to Apache. 
-Then, in a separate window, modify the code and push a commit. Run <code class="highlighter-rouge">git rebase -i HEAD~2</code> and 
+Then, in a separate window, modify the code and push a commit. Run <code class="language-plaintext highlighter-rouge">git rebase -i HEAD~2</code> and 
```

Comment:
I think these other changes are coming from differences in jekyll version - are you using 4.1.x? Not a big deal but I am not sure everyone is and I usually revert the noise changes and one day we'll standardize!

Reply:
I'm running Jekyll 4.1.1

Reply:
Ha OK then your results are 'correct'; it had been previously generated with jekyll 3.x. I think it's fine to choose to revert those, or commit them. Really we should regenerate everything with jekyll 4.1.x and really ask people stick to the latest, and be done with it.